scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -10,13 +11,29 @@
             <strong>Models</strong>: {{ gene.inheritance|join(',') }}
           </div>
         {% endif %}
+      </div>
+      ">
+        <a href="{{ url_for('genes.gene', hgnc_id=gene.hgnc_id) }}">
+          {{ gene.hgnc_symbol or gene.hgnc_id }}
+        </a>
+      </div>
+   
+    {% endfor %}
+  </div>
+  {% if variant.panels|count > 0 %}
+    <div
+       class="badge pull-right"
+       data-toggle="popover"
```

Comment:
Actually, when I think about it: why not use the tooltip framework as for the other hovers? That way you will get the same background color etc?

Reply:
The tooltips/popovers are already inconsistently used in the variants view. Both the compounds and comments hovers use popover (supposedly since they contain table elements?), and the others use tooltips. I guess the main difference (apart from the look) is that you can click popovers to make them stick.

Any guidelines on when to use one or the other? (if not, we should probably avoid bikeshedding this issue :))

Reply:
Thanks; I didn't notice.. :blush: It seems the intended use is popover for "new content" and tooltip for short remarks that really don't add anything. I guess that qualifies for gene names, not so sure about the frequency. At any rate, do go ahead and use popover, and we can later change the styles for either or all if we want it to look consistent!